app/controllers/users_controller.rb

```diff
@@ -1,21 +1,28 @@
 class UsersController < ApplicationController
   def show
-    # @user = User.find(params[:id])
-    # debugger
+
+  end
+
+  def new
+    @user = User.find_by(params[:id])
```

Comment:
thêm code xử lý trường hợp user bị nil nè 
`return if @user` 
`flash[:warning] = "User not found"`
`redirect_to root_path`